abp_rewrite.py

```
@@ -20,7 +20,8 @@ def doRewrite(files, args):
     os.path.join(baseDir, 'scripts', 'abprewrite.js'),
     '--arg', ' '.join(args)
   ] + files
-  return subprocess.check_output(command, env=env).replace('\r', '')
+  result = subprocess.Popen(command,  stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=env).communicate()[0]
```

Previously, in case of an error, the error message were printed to stderr (as stderr wasn't catched) and a CalledProcessError were raised resulting into a non-zero exit status.

Now with your change, in case of an error, the error message is silently ignore, as you catch and discard stderr and ignore the return status.

I suggest simply using `check_call` instead `check_ output`:

```python
def doRewrite(files, args):
  ...
  subprocess.check_call(command, env=env)

if __name__ ==  '__main__':
  ...
  doRewrite(...)
```

That way it's simpler, but errors will be still handled correctly. The only difference however would be that we might have windows-style new lines (`\r\n`) when running the code on Windows. But that shouldn't be a problem.